lib/active_shipping/shipping/carriers/ups.rb

```diff
@@ -45,15 +45,15 @@ class UPS < Carrier
           :occasional
         end
       end
-      
+
       DEFAULT_SERVICES = {
         "01" => "UPS Next Day Air",
-        "02" => "UPS 2nd Day Air",
+        "02" => "UPS Second Day Air",
```

Comment:
What is the reason behind this change? `UPS 2nd Day Air` is the actual name: https://www.ups.com/content/us/en/shipping/time/service/second_day.html